core/domain/feedback_services.py

```diff
@@ -172,38 +243,69 @@ def get_next_page_of_all_feedback_messages(
     return (result_messages, new_urlsafe_start_cursor, more)
 
 def get_thread_analytics_multi(exploration_ids):
-    """Returns a dict with feedback thread analytics for the given exploration
-    ids.
+    """Fetches all FeedbackAnalytics, for all the given exploration ids.
+
+    A FeedbackAnalytics contains the exploration id the analytics
+    belongs to, how many open threads exist for the exploration,
+    how many total threads exist for the exploration.
+
+    Args:
+        exploration_ids: list of str. A list of exploration ids.
+
+    Returns:
+        list of FeedbackAnalytics. It's in the the same order as the input
+        list. If the exploration id is invalid, the number of threads will
+        be zero in the corresponding FeedbackAnalytics object.
```

Comment:
This really is a nit, but "If the exploration id is invalid, the number of threads in the corresponding FeedbackAnalytics object will be zero." reads better.